modern-sass-compiler/assets/settings.scss

```diff
@@ -0,0 +1,5 @@
+/* DON'T USE @use here */
+@forward 'vuetify/settings' with (
```

Comment:
this should be `@use` when using it directly in the `styleConfig` and `@forward`  when adding `main.scss` (from vuetify docs, I need to review this)